sycl/doc/design/NonReusableDeviceCode.md

```diff
@@ -0,0 +1,45 @@
+# Non-Reusable Device Code
```

Comment:
The User Manual describes this mode as "non **relocatable** device code".  We should use consistent terms.

Reply:
sorry this is a typo, ill fix this. too many re- words